CMakeLists.txt

```diff
@@ -176,10 +175,8 @@ endif()
 
 if(UXRCE_BUILD_TESTS)
     ExternalProject_Add(itests
-        GIT_REPOSITORY
-            https://github.com/eProsima/Micro-XRCE-DDS-Integration-Tests.git
-        GIT_TAG
-            ${_tests_tag}
+        SOURCE_DIR
+            ${CMAKE_CURRENT_SOURCE_DIR}/test
```

Comment:
Maybe it is better to use `CMAKE_CURRENT_LIST_DIR`.

Reply:
Solved